Gameplay/ui/MainMenuManager.h

```diff
@@ -69,6 +69,10 @@ class MainMenuManager : public Script
 	const float _quit_button_delay_duration;
 	float _remaining_waiting_time_for_quit;
 	bool _started_to_quit;
+
+	SERIALIZE_FIELD(int, _option_selected);
```

Comment:
Do we need to serialize this bad boy?

Reply:
No, it's the fast way I found to debug it